weather@mockturtl/files/weather@mockturtl/po/sv.po

```diff
@@ -669,52 +678,161 @@ msgstr "Väder"
 
 #. metadata.json->description
 msgid "View your local weather forecast"
-msgstr "Se din lokala väderprognos"
-
-#. 3.6->settings-schema.json->head->description
-msgid "Settings for weather@mockturtl 3.6"
-msgstr "Inställningar för weather@mockturtl 3.6"
+msgstr "Visa din lokala väderprognos"
 
 #. 3.8->settings-schema.json->head->description
 msgid "Settings for weather@mockturtl 3.8"
 msgstr "Inställningar för weather@mockturtl 3.8"
 
 #. 3.8->settings-schema.json->dataService->options
+#. 3.4->settings-schema.json->dataService->options
 msgid "OpenWeatherMap (no key needed)"
 msgstr "OpenWeatherMap (ingen nyckel behövs)"
 
 #. 3.8->settings-schema.json->dataService->options
+#. 3.4->settings-schema.json->dataService->options
 msgid "DarkSky"
-msgstr "DarkSky"
+msgstr "Mörk himmel"
```

Comment:
I think the API name should not be translated

Reply:
you're right.